.cargo/config.toml

```diff
@@ -1,28 +1,27 @@
 [unstable]
-build-std = ["core"]
+build-std = ["core", "alloc"]
 build-std-features = ["compiler-builtins-mem"]
 
 [target.nanosplus]
 rustflags = [
-  "-C",
-  "codegen-units=1",
-  "-C",
-  "save-temps",      # Fixes incremental builds by keeping .bc and .ll files
-  "-C",
-  "embed-bitcode",
+  "-C", "codegen-units=1",
+  "-C", "save-temps", # Fixes incremental builds by keeping .bc and .ll files
+  "-C", "embed-bitcode",
+]
+
+[target.nanos]
```

Comment:
Did you mean to bring back the nanos target?

Reply:
No, this came directly from upstream, removing.